docs/moderator-handbook.md

```diff
@@ -190,7 +190,11 @@ Again, if you have any problems or questions, make a post with your actions in t
 
 ### Dealing with Posted Solutions
 
-When a user posts a solution to any of the questions, remove it and either, kindly let them know that solutions are not permitted on the forum in your own words or, use the **Solution Instead of Help** "Canned reply".
+When a user replies in a help thread specific to the freeCodeCamp curriculum with a solution, remove it and either, kindly let them know that solutions are not permitted in help threads in your own words or, use the **Solution Instead of Help** "Canned reply".
```

Comment:
```suggestion
When a user replies in a help thread specific to the freeCodeCamp curriculum with a solution, remove it and kindly let them know that solutions are not permitted in help threads in your own words or, use the **Solution Instead of Help** "Canned reply".
```

Reply:
Before I resolve this, you are talking about this text? 

> When a user replies in a help thread specific to the freeCodeCamp curriculum with a solution, remove it and kindly let them know that solutions are not permitted in help threads in your own words or, use the **Solution Instead of Help** "Canned reply".

Reply:
Yes correct. Before when the sentence had the word "and either," it was giving an option between a response that is made up vs a canned one. But now the option has shifted to become between removing and using a made up response OR using a canned response (no removal needed).

Reply:
make sense 👍